### PR TITLE
Remove "#doctest: +SKIP" comments on valid docstrings #6572

### PR DESCRIPTION
**Pull request by mickyaero**
# doctest: +SKIP ....changes

### CODE REVIEW

**Comment by thisch** [comment]
Note that the evalf skipped doctests are skipped on purpose.  Otherwise the integrationtests would run too long.

**Comment by thisch** [comment]
@mickyaero, I think you accidentally pushed your printing changes to the wrong branch. The final commit of this PR should be c38e5ff.

**Comment by mickyaero** [comment]
@asmeurer 

All the tests are passing!
Are there any other changes I have to do regarding #10035 ??

File: sympy/printing/str.py [hunk 1]
```diff
@@ -741,6 +741,20 @@ def _print_Tr(self, expr):
...
+        return "S.Naturals"
+
+    def _print_Complexes(self, p):
+        return "S.Complexes"
+
+    def _print_EmptySet(self, p):
+        return "S.EmptySet"
+
+    def _print_Naturals0(self, p):
+        return "S.Naturals0"
```
Comment (moorepants) [inline sympy/printing/str.py]: Can you justify this change?

**Comment by moorepants** [comment]
@asmeurer Seems like these are fixed according to your comments in #6572.

File: doc/src/modules/solvers/solveset.rst [hunk 2]
```diff
@@ -256,7 +256,7 @@ What is this domain argument about?
     >>> solveset(x**2 + 1, x) # domain=S.Complexes is default
     {-I, I}
     >>> solveset(x**2 + 1, x, domain=S.Reals)
-    EmptySet()
+    S.EmptySet
```
Comment (gxyd) [inline doc/src/modules/solvers/solveset.rst]: Can you please explain why these changes are required?

**Comment by gxyd** [comment]
Closing this in favor of #13510
